crates/kubelet/src/node.rs

```diff
@@ -4,8 +4,34 @@ use k8s_openapi::api::coordination::v1::Lease;
 use k8s_openapi::api::core::v1::Node;
 use k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
 use kube::api::{Api, DeleteParams, PatchParams, PostParams};
+use kube::Error;
 use log::{debug, error, info};
 
+macro_rules! retry {
+    ($action:expr, times: $num_times:expr, $on_err:expr) => {{
+        let mut n = 0u8;
+        let mut duration = std::time::Duration::from_millis(100);
```

Comment:
For maximum flexibility, this should probably be a configurable value. Not a blocking request though